cmd/witness/cmd/sign.go

```diff
@@ -47,6 +47,12 @@ func SignCmd() *cobra.Command {
 //we need to abstract where keys are coming from, etc
 func runSign(so options.SignOptions) error {
 	ctx := context.Background()
+
+	if so.KeyOptions.FulcioURL != "" {
```

Comment:
we shouldn't add the option to sign for fulcio if it's not supported

Reply:
we can sign, just cant verify yet!

Reply:
It looks like we’re throwing an error if witness run is run with the fulcio url option, am I misreading it?

Reply:
Only for sign.  Fulcio options are in the KeyOptions struct.  We can add support for signing with fulcio, but we also need ca verification of policy for this to make sense.